openfisca_core/entities.py

```diff
@@ -71,7 +71,7 @@ def init_variable_values(self, entity_object, entity_id):
 
             if not isinstance(variable_values, dict):
                 raise SituationParsingError(path_in_json,
-                    'Invalid type: must be of type object. Input variables must be set for specific periods. For instance: {"salary": {"2017-01": 2000, "2017-02": 2500}}')
+                    u'Invalid type: must be of type object. Input variables must be set for specific periods. For instance: {"salary": {"2017-01": 2000, "2017-02": 2500}}')
```

Comment:
When tested with curl, got : 
```
 For instance: {\"salary\": {\"2017-01\": 2000, \"2017-02\": 2500}}"
```
I was expecting : 
```
 For instance: {"salary": {"2017-01": 2000, "2017-02": 2500}}"
```

Reply:
`For instance: {\"salary\": {\"2017-01\": 2000, \"2017-02\": 2500}}` is correct as it is included in a JSON response like: 
```
{
  "persons": {
    "bill": {
      "salary": "Invalid type: must be of type object. Input variables must be set for specific periods. For instance: {\"salary\": {\"2017-01\": 2000, \"2017-02\": 2500}}"
    }
  }
}
```